how-to/how-to-use-kata-containers-with-nemu.md

```diff
@@ -0,0 +1,59 @@
+# Kata Containers with NEMU
+Kata Containers relies by default on Qemu hypervisor in order to spawn the virtual machines running containers. [NEMU](https://github.com/intel/nemu) is a fork of Qemu focused on increasing the security through 3 different ways:
+- Reducing the number of lines of code.
+- Removing all legacy devices.
+- Reducing the emulation as far as possible.
+
+As the goal for Kata Containers is to increase the security, this is a logical step to get Kata relying on NEMU as its hypervisor.
+
+## How to get started
+
+This section will describe how to run Kata Containers with NEMU, first by explaining how to download, build and install it. Then this document will walk through the steps needed to update your Kata Containers configuration in order to run with NEMU. 
+
+### Pre-requisites
+This document requires Kata Containers to be installed in your system.
+
+### NEMU
+___Download, build and install___
+```bash
+$ git clone https://github.com/intel/nemu.git
+$ cd nemu
+$ git fetch origin/master
+$ git checkout experiment/automatic-removal
+$ SRCDIR=$PWD ./tools/build_x86_64_virt.sh
+```
+After those commands successfully returned, you will find the NEMU binary at `$HOME/build-x86_64/x86_64_virt-softmmu/qemu-system-x86_64_virt`.
+
+You also need the `OVMF` firmware in order to boot the virtual machine's kernel. It can currently be found at this [location](https://github.com/rbradford/edk2/releases/). Please choose the latest version of this firmware (`v0.2` in this example):
+```bash
+$ mkdir -p $HOME/nemu_firmware
+$ wget https://github.com/rbradford/edk2/releases/download/0.2/OVMF.fd -P $HOME/nemu_firmware/
+```
+
+### Configure Kata Containers
+All you need from this section is to modify the configuration file `configuration.toml`, to specify the options related to the hypervisor.
+___`/usr/share/defaults/kata-containers/configuration.toml`___
+```toml
```

Comment:
The ellipsises are giving errors as they are not valid TOML. I think this might be clearer as a diff. Something like:

```diff
 [hypervisor.qemu]
-path = "/usr/bin/qemu-lite-system-x86_64"
+path = "/home/foo/build-x86_64/x86_64_virt-softmmu/qemu-system-x86_64_virt"
 kernel = "/usr/share/kata-containers/vmlinuz.container"
 initrd = "/usr/share/kata-containers/kata-containers-initrd.img"
 image = "/usr/share/kata-containers/kata-containers.img"
-machine_type = "pc"
+machine_type = "virt"
 
 # Optional space-separated list of options to pass to the guest kernel.
 # For example, use `kernel_params = "vsyscall=emulate"` if you are having
@@ -31,7 +31,7 @@
 
 # Path to the firmware.
 # If you want that qemu uses the default firmware leave this option empty
-firmware = ""
+firmware = "/home/foo/nemu_firmware/OVMF.fd"
 
 # Machine accelerators
 # comma-separated list of machine accelerators to pass to the hypervisor.
```

Reply:
Yes I like that, it renders definitely better.